.github/workflows/tests.yml

```diff
@@ -41,20 +43,25 @@ jobs:
           - windows
         python:
           - '3.7'
+          - '3.8'
+          - '3.9'
+          - '3.10'
           - '3.11'
+          - '3.12-dev'
+          - 'pypy-3.9'
```

Comment:
```suggestion
          - '3.12'
          - 'pypy-3.10'
```

Reply:
And then below add the last line to:
```diff
  name: Set up target Python
        uses: actions/setup-python@v4
        with:
          python-version: ${{ matrix.python }}
+         allow-prereleases: true
```
https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#allow-pre-releases

Reply:
Why do you suggest to drop testing with pypy-3.9 and test pypy-3.10 instead?

Reply:
To ensure that there are not any compatibility issues.

Reply:
Do you think that `pypy-3.9` was picked at random? Why is it more important to test with `pypy-3.10` than with `pypy-3.9`? Is anything in the PyPy 3.10 release changelog making you believe that it is important to test with this release?